docs/backends/quickwit.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "Quickwit"
+---
+
+## Configuring Quickwit Backend
+
+The only required field is the Quickwit endpoint which is supporting OTLP grpc (it remain the same http endpoint used for the Quickwit RESTful API).
```

Comment:
I think it can be useful to also mention that the gRPC endpoint should be unencrypted, so users will not be confused and try to use the destination where it is not supported.

Reply:
Yeah and I think I was wrong here, Quickwit support two ports: one for the RESTful API (7280) and the GRPC OTLP (7281). So I'll fix this part

Reply:
Done